.github/workflows/publish-package.yml

```diff
@@ -21,11 +21,14 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: 3.8
-    - name: Install pep517 and twine
+    - name: Install pep517, check-manifest, and twine
       run: |
         python -m pip install --upgrade pip setuptools wheel
         python -m pip install pep517 --user
-        python -m pip install twine
+        python -m pip install check-manifest twine
```

Comment:
can we migrate this into a [publish] part of the setup.py?

Reply:
We could, but then we'd also need to install `pyhf` as well to use the extras API with `pip` (right?). Seems a bit unnecessary to install multiple things when we just need 2. Though I guess the argument for it is that another 10 seconds doesn't matter one way or another?